tests/unit/crt/auth/test_crt_signers.py

```diff
@@ -0,0 +1,34 @@
+import botocore
+
+from tests import requires_crt
+from tests.unit.auth.test_signers import (
+    TestS3SigV4Auth, TestSigV4Presign, TestSigV4Resign
+)
+
+
+@requires_crt
+class TestCrtS3SigV4Auth(TestS3SigV4Auth):
+    # Repeat TestS3SigV4Auth tests, but using CRT signer
+
+    def setUp(self):
+        AuthClass = botocore.crt.auth.CrtS3SigV4Auth
```

Comment:
Is there any reason to to this as part of `setUp` and not just as part of the class definition like we did for the super class?